app/utils/utils.js

```diff
@@ -26,6 +32,38 @@ const getPrivateKeyAddress = (network, privateKey) => {
   return privateKey.address;
 };
 
+const mnemonicToPrivateKey = (mnemonic) => {
+	const seed = bip39.mnemonicToSeed(mnemonic)
+
+	const master = bip32.fromSeed(seed)
+  const child = master.derivePath(`m/44'/5757'/0'/0/0`)
```

Comment:
This should probably use the `app/common/constants.js` constant `PATH` rather than copying the path over.